frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -545,4 +558,34 @@ export function applyState(state: PartialUrlManagerState, ignoreZoom: boolean =
   if (state.rotation != null) {
     Store.dispatch(setRotationAction(state.rotation));
   }
+  if (state.activeMappingByLayer != null) {
+    for (const layerName of Object.keys(state.activeMappingByLayer)) {
```

Comment:
TODO Clean up this part of the code:

- [ ] Error handling
- [ ] What if the layer with a mapping is not activated?